src/components/VDataIterator/VDataIterator.js

```diff
@@ -49,38 +49,24 @@ export default {
       return this.$createElement(this.contentTag, data, children)
     },
     genEmptyItems (content) {
-      return this.$createElement('div', {
+      return [this.$createElement('div', {
         'class': 'text-xs-center',
         style: 'width: 100%'
-      }, content)
+      }, content)]
     },
     genFilteredItems () {
+      if (!this.$scopedSlots.item) {
+        return null
+      }
+
       const items = []
-      this.filteredItems.forEach((item, index) => {
+      for (let index = 0, len = this.filteredItems.length; index < len; ++index) {
```

Comment:
https://jsperf.com/native-map-versus-array-looping

Native loop outperforms by a good bit in most cases, and by a lot in some cases (like latest safari on ios)